Tasks/UsePythonVersionV0/Tests/L0.ts

```diff
@@ -38,9 +40,10 @@ describe('UsePythonVersion L0 Suite', function () {
         assert(didPrependPath(testRunner, pythonBinDir));
         assert.strictEqual(testRunner.stderr.length, 0, 'should not have written to stderr');
         assert(testRunner.succeeded, 'task should have succeeded');
+        done();
```

Comment:
Do we need to use done here? There doesn't seem to be any asynchronous code

Reply:
Thanks! Resolved this.